composer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "autoload": {
+    "psr-4": {
+      "Animals\\": "src/Animals"
+    }
+  }
+}
```

Comment:
I've made this file manually. Must I make an empty line on the end of every file? Is it some standard or something else? I haven't seen any notes about this before. 
I will do those empty lines in future.